src/lob-address-elements.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import { parseWebPage } from './form-detection.js';
+import { findElm, findForm, findValue, findPrimaryAddressInput, parseWebPage } from './form-detection.js';
```

Comment:
I like your reorganization to move this methods to the separate DOM traversal/find module. Nice approach.